### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by benoftoolofna**
initial coverage.

### CODE REVIEW

File: bin/init.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ module.exports = () => {
 		})
 }
 
-const setProjectID = () => {
+exports.setProjectID = () => {
```
Comment (positlabs) [inline bin/init.js:37]: What's the benefit of using exports here?

Comment (benoftoolofna) [reply bin/init.js:37]: will be adding more testing coverage in the future.  can't test private methods cleanly.  nothing dangerous is exposed here

Comment (positlabs) [reply bin/init.js:37]: If you need to expose these via require statements, I think you have to reassign exports to module.exports (e.g. module.exports = exports). Or does this already work as intended?

Comment (positlabs) [reply bin/init.js:37]: Nevermind. That's only true if the value of exports is reassigned. 
> However, be aware that like any variable, if a new value is assigned to exports, it is no longer bound to module.exports

**Pull request merged**